### DISCUSSION TITLE
tract engine memory footprint

### DISCUSSION THREAD

**Discussion by flo-th** [sonos/tract#947] [General]
Hi all,

Could you tell me what is the memory  footprint of tract engine ? I would like to embedd it on MCU.

Thanks in advance,

**Comment by kali** [comment]
tract is not MCU-sized. It's designed for small CPU (armv7-a, armv8-a...). It needs the stdlib, it does allocation absolutely everywhere...

**Reply by flo-th** [reply]
My bad, wrong copy/paste.
I meant ARM CPU (in particulary V8 model) and not MCU.
